README.md

```diff
@@ -1328,6 +1328,7 @@ Software that does not fit in another section.
 - [DomainMOD](https://domainmod.org) - Application to manage your domains and other internet assets in a central location. DomainMOD includes a Data Warehouse framework that allows you to import your WHM/cPanel web server data so that you can view, export, and report on your data. ([Demo](https://demo.domainmod.org), [Source Code](https://github.com/domainmod/domainmod)) `GPL-3.0` `PHP`
 - [DOMJudge](https://www.domjudge.org/) - A system for running a programming contest, like the ICPC regional and world championship programming contests. ([Demo](https://www.domjudge.org/demo), [Source Code](https://github.com/DOMjudge/domjudge)) `GPL-2.0/BSD-3-Clause/MIT` `PHP`
 - [F-Droid](https://f-droid.org) - Server tools for maintaining an F-Droid repository system. ([Source Code](https://gitlab.com/fdroid/fdroidserver)) `AGPL-3.0` `Python/Docker`
+- [Flagsmith](www.flagsmith.com) - We help teams ship faster and continuously improve digital products with feature flags. Alternative to LaunchDarkly. ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD3` `python`
```

Comment:
```suggestion
- [Flagsmith](www.flagsmith.com) - We help teams ship faster and continuously improve digital products with feature flags. Alternative to LaunchDarkly. ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD-3-Clause` `Docker`
```

Reply:
>Furthermore, I have a question regarding the presence of a secondary license mentioned in [api/permissions/LICENSE.md](https://github.com/Flagsmith/flagsmith/blob/main/api/permissions/LICENSE.md). Could you clarify if this license is a custom one? What specific purpose does it serve? From my interpretation, it appears to ensure that users purchase additional licenses when they require additional features. Is my understanding correct?

Yes, this license has been customised. It prevents unauthorised used of the role based access functionality, for which some of the source code resides in the open source repository. This does not prevent users from deploying or using the software, it simply limits this one specific area of functionality.

Reply:
I have applied the suggested changes.